packages/channels/src/telegram/config.ts

```diff
@@ -0,0 +1,9 @@
+import Joi from 'joi'
+
+export interface TelegramConfig {
+  botToken: string
+}
+
+export const TelegramConfigSchema = Joi.object({
+  botToken: Joi.string().required()
+}).options({ stripUnknown: true })
```

Comment:
It's likely we'll change the names of these config options to standarize it accross channels. For now we can merge this PR with that and standardize it later.